### PR TITLE
Borrower profile page init

### PR DESCRIPTION
**Pull request by dyersituations**
- Merging in the borrower profile initial load performance branch
- This code has been individually reviewed + merge conflicts resolved with other recent work
- Putting up this PR to get a final set of reviews before merging into `main`

Overview:
- Removed 2 experiments
- Moved `LoanBookmark` prefetched data to `mounted`
- Moved some `Summary Card` prefetched data to `mounted`

New unreviewed changes:
- Removed unneeded try catch from `SummaryCard` `mounted` async call
- Merge with recent BP changes in `main`

image

### CODE REVIEW

**Review by mcstover — APPROVED** [review]
It will be interesting to see how this affects our page load speed.

File: src/components/BorrowerProfile/LendCta.vue [hunk 1]
```diff
@@ -558,7 +516,7 @@ export default {
 			this.matchingTextVisibility = this.status === 'fundraising' && this.matchingText && !this.isMatchAtRisk;
 
 			if (this.status === 'fundraising' && this.numLenders > 0) {
-				this.lenderCountVisibility = !this.socialExpEnabled;
+				this.lenderCountVisibility = true;
```
Comment (emuvente) [inline src/components/BorrowerProfile/LendCta.vue:519]: This should be updated on line https://github.com/kiva/ui/pull/4458/files#diff-09ca3cc753df5d76e7f1404c47aa4ee2f07ce9ced29b352f4739cce2c1db3168R619 as well

**Review by emuvente — APPROVED** [review]
I love these changes! Separating the fundraising progress from the prefetch will give us a performance boost and unlock more opportunities for performance gains (like caching fields at the graphql server level for loan properties that don't change). Fundraising progress is always going to be uncachable and relatively expensive to request, so separating that into a client-only query is definitely the way forward!

And thank you for cleaning up those experiments! I only saw one change needed related to recent changes to the lend cta.

File: src/components/BorrowerProfile/LendCta.vue (hunk 1 above)
Comment (dyersituations) [reply src/components/BorrowerProfile/LendCta.vue:519]: Good call. I had to redo the merge and missed this on the second go. That link doesn't seem to go to a line, but I'm assuming you were referring to line 619 (see below), and we just need to remove the `!this.socialExpEnabled`.

Old
```
if (this.status === 'fundraising' && this.numLenders > 0 && !this.socialExpEnabled) {
```

New
```
if (this.status === 'fundraising' && this.numLenders > 0) {
```

Comment (dyersituations) [reply src/components/BorrowerProfile/LendCta.vue:519]: Change pushed, and I'll go ahead and merge. Thanks!

**Pull request merged**